<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>321</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AccountEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SurePasswordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="AddressButton_2">
        <property name="text">
         <string>个人偏好</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PreferenceEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="AddressButton">
        <property name="text">
         <string>地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AddressEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>卖家请填写店名</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>店名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StoreNameEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="UpdateButton">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SureButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ReturnButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
